### PR TITLE
fix(monitoring): restore was failing to file dashboard file

### PR DESCRIPTION
**Pull request by fruch**
since #4627 the dashboard files now has random part to them
and the `show-monitor` command was failing

this fix only getting the first dashboard and doesn't handle
the case of restoring multiple dashboards (for operator multi
tenant runs)

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by vponomaryov** [comment]
I see that show-monitor job works ok as of now. Where do you see errors?
The only problem for general case that I can see now is that the default path has no substituted value for the "$test_name" variable.

Then, we should not create each dashboard we find because they are needed only for population of `different` monitoring nodes, not the same one where we have the same `db_cluster node set`.

So, are you sure about the current state of the fix?

I would expect to see the creation of separate monitoring nodes with it's own db_cluster and just 1 SCT dashboard, the correct one. In other words, fix is needed for the K8S multi-tenant case, not the general one.

**Comment by fruch** [comment]
this fix is need, since now the name was changed, and the show-monitor was looking for "scylla-dash-per-server-nemesis.{}.json" which isn't the name anymore.

File: sdcm/monitorstack/__init__.py [hunk 1]
```diff
@@ -282,45 +283,43 @@ def run_monitoring_stack_containers(monitoring_stack_dir, monitoring_data_dir, s
...
-        dashboard_config = json.load(f)
-        # NOTE: remove value from the 'dashboard.id' field to avoid following error:
-        #
-        #       Error message {"message":"Dashboard not found","status":"not-found"}
-        #
-        #       Creating dashboard from scratch we should not have 'id', because Grafana will try
-        #       to 'update' existing one which is absent in such a case.
-        if dashboard_config.get('dashboard', {}).get('id'):
-            dashboard_config['dashboard']['id'] = None
+    for sct_dashboard_file in sct_dashboard_files:
```
[the remark this answers is not in the document]
Comment (fruch) [reply sdcm/monitorstack/__init__.py]: I've removed the loop, and also removed the usage of scylla_version, we don't really need it, we just take the dashboard we have. (and once you'll get to it, all the dashboards)

**Review by vponomaryov — APPROVED** [review]

**Comment by vponomaryov** [comment]
LGTM
@fruch Hope you have also tested it.

**Comment by fruch** [comment]
> LGTM @fruch Hope you have also tested it.

yes I did

**Pull request merged**